curriculum/challenges/english/21-a2-english-for-developers/learn-introductions-in-an-online-team-meeting/657ca36eacc797817451ece6.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 657ca36eacc797817451ece6
+title: Task 27
+challengeType: 19
+dashedName: task-27
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Bob: Can you hand me that file, please? I need to review it before our meeting.
```

Comment:
This has passed me by in the first review. This line is not in the dialogues, so there's no audio for that.